tests/store/test_sqlalchemy_store.py

```diff
@@ -545,6 +545,31 @@ def test_set_tag(self):
         run = self.store.get_run(run.info.run_id)
         self.assertTrue(tkey in run.data.tags and run.data.tags[tkey] == new_val)
 
+    def test_delete_tag(self):
+        run = self._run_factory()
+
+        k0, v0 = 'tag0', 'val0'
```

Comment:
@ankitmathur-db Can we also test the case where we call `set_tag()` with the same key on two different runs and ensure that deleting the tag from one run does not delete the tag from the other (per the filter issue you mentioned yesterday)